### ISSUE TITLE
Mind Control/Dominate Mind effects should be dispellable (Need Info)

### ISSUE THREAD

**Issue by ohganot** [elysium-project/server#2597]
Lucifron adds cast a spell called Dominate Mind which should be dispellable.

Multiple guides from 2005 state that it should be dispelled as fast as possible:
http://www.asga.co.uk/forum/index.php?topic=77.0
http://www.tentonhammer.com/guides/mc-boss-guide-boss-1-lucifron

This might also apply to Mind Controls in general, but need more research done for that.
Patch 1.4.0 notes state that:
"Charm and Polymorph mechanics can now be reactively countered by spells and abilities (e.g. Druid Shapeshifting will now cancel Polymorph effects)."

**Comment by GuybrushGit** [comment]
So when using dispel on an MCed friend, it can dispell the MC debuff in adition to dispelling a random buff?

Does the MC debuff have dispell priority? Do you have to spam dispell and remove random buffs from him until you get lucky and remove his MC?

**Comment by Cabro** [comment]
If i remember something vaguely about this, is that paladins couldn't dispel MC because it was a hostile spell, while priests and shamans could.
This mechanic of dispelling while MC effects is a special one and should be dealt thoroughly.

**Comment by ohganot** [comment]
Yeah, dispells worked differently on MCd targets. From what I understand, dispelling a mind controlled target would only dispel the mind control, but only offensive dispells would work (Purge, Dispel Magic, Devour Magic).
The video is quite clear for the Lucifron add Dominate Mind spell, but I'm not 100% sure how it should work for priest Mind Control.

**Comment by Cabro** [comment]
As far as i remember, it all depended on the faction of your MC target. As in, if your target was same faction as you and was MC, it would dispel the MC if you were priest, shaman or warlock pet. If you were paladin, nothing dispelled.
If it was opposite faction, it would dispel his beneficial buffs (stamina, intellect etc). Again, not if you were a paladin.

**Comment by Fiskehatt** [comment]
## Some extra evidence:

Lucifron - Molten Core
http://www.ownedcore.com/forums/world-of-warcraft/world-of-warcraft-guides/1004-molten-core-dummies.html (2006-06-10)

> First, the guards randomly Mind Control people. Priests need to dispel this ASAP, but because MC'ed players tend to use their long-duration abilities, WARRIORS ABSOLUTELY MUST BLOW THEIR INTIMIDATING SHOUT BEFORE THE BATTLE. If they do not, they risk getting controlled, running into the middle of the raid, and fearing everyone.

Lady Illucia Barov - Scholomance
http://www.tentonhammer.com/guides/scholomance-guide (2005-11-05)

> Lady Illucia Barov can mind control one of your party,
if so, remember that both Horde and Alliance can deal with this, Shamans can
purge this, and Priests dispel it as it's a magic buff or if your ally is low on health you can polymorph them to let them regen their health quickly.

https://www.gamefaqs.com/pc/534914-world-of-warcraft/faqs/37409 (2005-08-10)

> Lady Illucia Barov has a tendency to mind
control one of your party, I believe that Shamans can purge this, and priests
dispel it as it's a magic buff.

Balnazzar - Stratholme 
http://www.tentonhammer.com/guides/guide-to-stratholme (2005-08-05)

> When fighting him in demon form he will mind control players several times throughout the fight. An easy way to break this is the have the mage sheep the mind controled player and then have a priest dispel magic on them. Overall he is not a hard fight though as long as you keep him aggroed on the tank (sheep / dispel the tank fast if he is MC'ed) and do steady damage.

Like Purge, Dispel and Devour Magic, I think that [Shield Slam](http://db.vanillagaming.org/?spell=23922) should be able to dispel MC:

> Slam the target with your shield, causing 225 to 236 damage, modified by your shield block value, and has a 50% chance of **dispelling 1 magic effect** on the target. Also causes a high amount of threat.

**Comment by GuybrushGit** [comment]
Yeah its pretty obvious that it should dispel the only doubt I have is if it has priority over buffs.

**Comment by Fiskehatt** [comment]
At least in the videos linked by you and @ohganot, the MC gets dispelled first. I can think of two alternative reasons for this:

1. MC has dispel priority over other buffs and debuffs.
2. It is the most newly applied of all magic buffs and debuffs. It is difficult to confirm or disprove this based on the videos because, in them, no one puts any buffs or debuffs on the players after being MC'd. 

If the case 2 is correct, you could basically stop people from dispelling MC by putting magic buffs and debuffs on mindcontrolled people.
